recipes-core/images/files/partition.txt

```diff
@@ -3,6 +3,6 @@ label-id: 0xa4b54066
 device: /dev/mmcblk2
 unit: sectors
 
-/dev/mmcblk2p1 : start=       20480, size=      204800, type=c, bootable --label boot
```

Comment:
This file will have to be made machine specific and generated using a recipe, we will discuss this